src/developer_config.yaml

```diff
@@ -19,15 +19,17 @@ hdfs_paths:
   snapshot_path: "/ons/rdbe_dev/snapshot-202012-002-fba5c4ba-fb8c-4a62-87bb-66c725eea5fd.json"
   masterlist_path: "data/external/ONSPD_NOV_2022_UK.csv"
   postcode_masterlist: "data/external/ONSPD_NOV_2022_UK.csv"
+  mapper_path: "/ons/rdbe_dev/BERD_Mappers/SIC07_to_PG_Conversion-From_2016_Data.csv"
   history_path: "/ons/rdbe_dev/BERD_V7_Anonymised"
 network_paths:
   root: None
   logs_foldername: "logs/run_logs"
   snapshot_path: "R:/BERD Results System Development 2023/DAP_emulation/survey_return_data/snapshot-202212-002-83b5bacd-7c99-45cf-b989-d43d762dd054.json"
   postcode_masterlist: "R:/BERD Results System Development 2023/DAP_emulation/ONS_Postcode_Reference/ONSPD_NOV_2022_UK.csv"
+  mapper_path: "R:/BERD Results System Development 2023/DAP_emulation/mappers/SIC07 to PG Conversion - From 2016 Data .csv"
```

Comment:
Suggestion:
Could you change the name of the file to get rid of the space before the  .csv? I think that might cause errors. And maybe give it the same name as the one on hdfs for consistency?